contracts/src/Rollup.sol

```diff
@@ -126,20 +144,18 @@ contract Rollup is RollupBase {
             block.number // deadline (unchallengeable)
         );
         emit AssertionCreated(lastCreatedAssertionID, msg.sender, _initialVMhash);
-
-        __RollupBase_init();
```

Comment:
I moved `__RollupBase_init` to the top of the function since we need to initialize the admin role, before we can add roles and initialize the initial validator whitelist.